herd/ASLSem.ml

```diff
@@ -673,7 +665,7 @@ module Make (Conf : Config) = struct
     let do_write_memory (ii, poi) addr_m datasize_m value_m an aexp acc =
       let* addr = M.as_addr_port addr_m
       and* datasize = datasize_m
-      and* value = value_m in
+      and* value = value_m >>= to_aarch64_val in
```

Comment:
I see, all ~~events~~ effects now have AArch64 style values (_.i..e._ no bitvectors). I did not dare do that. It looks like a good move.